build/Build.cs

```diff
@@ -15,13 +15,17 @@ partial class Build : NukeBuild
     [Parameter("Platform to build - x86 or x64. Default is 'x64'")]
     readonly MSBuildTargetPlatform Platform = MSBuildTargetPlatform.x64;
 
-    [Parameter($"Docker containers type to be used. One of '{ContainersNone}', '{ContainersLinux}', '{ContainersWindows}'. Default is '{ContainersLinux}'")]
+    [Parameter($"Docker containers type to be used. One of '{ContainersNone}', '{ContainersLinux}', '{ContainersWindows}', '{ContainersWindowsContainerTestsOnly}'. Default is '{ContainersLinux}'")]
     readonly string Containers = ContainersLinux;
 
+    [Parameter("TargetFramework to be tested. Default is empty, meaning all TFMs supported by each test")]
+    readonly TargetFramework TestTargetFramework = TargetFramework.NOT_SPECIFIED;
+
     const string ContainersNone = "none";
     const string ContainersAny = "any";
     const string ContainersLinux = "linux";
     const string ContainersWindows = "windows";
+    const string ContainersWindowsContainerTestsOnly = "windows-container-tests-only";
```

Comment:
nit: how about renaming to

```suggestion
    const string ContainersWindowsOnly = "windows-only";
```

Reply:
8760e6f45